building-a-component/readme.md

```diff
@@ -222,7 +221,31 @@ If either of these conditions are true, it is probably better to split it into i
 
 
 ## Contextual styles
-It is best to avoid contextual styles (components "reaching into" other components) — use variations instead.
+It is best to avoid contextual styles (high-level components "reaching into" other high-level components) — use modifiers instead.
+
+```scss
+.blog {
+  // ...some styles...
+}
+
+.blog-post {
+  // ...some styles...
+}
+
+// Good!
+//
+.blog-post--more {
```

Comment:
I think a better example for this could be something like
```
.blog-post--home {
}
```
and 
```
.home-page {
  .blog-post {
  }
}
```
What do you think?